README.md

```diff
@@ -1,2 +1,24 @@
-# malls-parser-skill
-Skill for parsing store name and location or a requested product/store
+# <img src='https://0000.us/klatchat/app/files/neon_images/icons/neon_skill.png' card_color="#FF8600" width="50" style="vertical-align:bottom">{title}
+## Summary
+Skill for mall parsing
+
+## Description
+{Skill parses mall web page and returns user name, location and work hours of requested shop/store}
+
+## Examples
+{Where is apple? Where can I find ABC stores?}
+
+## Contact Support
+Use the [link](https://neongecko.com/ContactUs) or [submit an issue on GitHub](https://help.github.com/en/articles/creating-an-issue)
+
+## Credits
+
+[NeonGeckoCom](https://github.com/NeonGeckoCom)
+{author_name}
```

Comment:
You can add yourself to the credits here in place of `{author_name}`
`[NeonMariia](https://github.com/neonmariia)`